hugegraph-server/hugegraph-postgresql/src/main/java/org/apache/hugegraph/backend/store/postgresql/PostgresqlTable.java

```diff
@@ -162,7 +161,7 @@ public PgWhereBuilder(boolean startWithWhere) {
         @Override
         protected String escapeAndWrapString(String value) {
             if (value.equals("\u0000")) {
-                return "\'\'";
+                return "''";
```

Comment:
better add a test or test it in local env first? (at least maybe add a PR comment here)